commit f8c8f173bce24009e86f9bb2f1de179c6f258865
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-08-06T18:03:45+01:00 (9 weeks ago)
    
    OS-7770 remove platform build dep on download.joyent.com
    Reviewed by: John Levon <john.levon@joyent.com>

diff --git a/build.sh b/build.sh
index 2560048..f9248da 100755
--- a/build.sh
+++ b/build.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 for dir in seabios vgabios kvm/test; do
@@ -22,7 +22,7 @@ if [[ "$PRIMARY_COMPILER_VER" -gt 4 ]]; then
 fi
 
 if [[ ! -d ${PNGDIR} ]]; then
-	(curl -k https://download.joyent.com/pub/kvm-cmd/libpng-1.5.4.tar.gz | \
+	(curl -k https://us-east.manta.joyent.com/Joyent_Dev/public/releng/kvm-cmd/libpng-1.5.4.tar.gz | \
 	    gtar -zxf -)
 	if [[ $? != "0" || ! -d ${PNGDIR} ]]; then
 		echo "Failed to get libpng."
